src/provider.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
```

Comment:
I think we have a provider helper in `fusion-react` that you could maybe use here